src/components/_global/BalTable/BalTable.vue

```diff
@@ -106,31 +106,61 @@
           >
             <td
               v-for="(column, columnIndex) in filteredColumns"
+              @click="navigate"
+              :href="href"
```

Comment:
Where are `navigate` and `href` coming from? I don't see any reference to them throughout the component? Also, why is `href` on a td tag?

Reply:
Forgot to commit the change on this, will update in a moment